public/css/customFonts.css

```diff
@@ -76,3 +76,7 @@ a:hover .tweet.fullname {
 #addSearchTerm:-ms-input-placeholder {
     color:    #999;
 }
+
+.ui.slider.cache.checkbox input:checked~.box:before, .ui.slider.cache.checkbox input:checked~label:before {
+  background-color: #a333c8!important
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2370125340/issues/source?bid=3148782&fileBranchId=3222074#l81)